### PR TITLE
Add image module and update images

### LINKED ISSUE #4211: Logo section outdated on homepage
Add image_template module and use logo section on logos on homepage

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Remove slack from list on homepage
- Add image module and update logo section on homepage to use logo section pattern

Fixes #4211 

## QA

- Open [demo](insert-demo-url)
- Check the logo section looks ok on the home page

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [x] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical-web-and-design/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [x] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [what's new page](https://github.com/canonical-web-and-design/vanilla-framework/blob/main/templates/docs/whats-new.md).
- [x] Documentation side navigation should be updated with the relevant labels.

### CODE REVIEW

**Comment by bartaz** [comment]
Linkchecker is failing, seems it has problems parsing cloudinary image links.
I checked ubuntu.com and it seems they have some config for linkchecker to ignore those: 

https://github.com/canonical-web-and-design/ubuntu.com/blob/99d51925933ce6885c8b646ca1aafc109d936e5a/.github/workflows/live-links.yaml#L28-L35

We should try to replicate that in our environment.

File: templates/_layouts/_root.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
     <title>{% block title %}{{ title }}{% endblock %}{% if self.title() %} | {% endif %}Vanilla documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script src="https://assets.ubuntu.com/v1/703e23c9-lazysizes+noscript+native-loading.5.1.2.min.js" defer></script>
```
Comment (bartaz) [inline templates/_layouts/_root.html:17]: Currently we have some lazy loading script in `templates/_layouts/site.html`, so I think we should keep it there (and remove the old one that uses Blazy).

**Comment by bethcollins92** [comment]
Woooo @bartaz checks have passed

**Review by bartaz — APPROVED** [review]
LGTM

**Pull request merged**